doc/nrf/ug_zigbee_qsg.rst

```diff
@@ -81,14 +81,18 @@ If you want to learn more about Zigbee topics and terminology mentioned in this
 Requirements
 ************
 
-For this quick start guide, you need three development kits of the following types:
+For this quick start guide, you need **three** development kits of the following types:
```

Comment:
Nit: although there is no prohibition for using bold, we usually reserve bold for some very specific use cases (https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Typographical+conventions ), so you might want to consider using italics here (with single asterisks).